app/controllers/projects/samples/attachments_controller.rb

```diff
@@ -28,12 +28,23 @@ def create
       end
 
       def destroy
-        authorize! @project, to: :update_sample?
+        authorize! @project, to: :destroy?
```

Comment:
We should use the `:destroy_sample?` in the project policy for this authorization

Reply:
Fixed in d9f8225c5d145849cec6a8481c7897234d4b8b2d

Reply:
> Sorry Chris I made a mistake here, we should change this back to `update_sample?`. The `destroy_sample?` authorization method only allows a member who is an owner to delete attachments, but we want to allow maintainers to delete attachments as well

I believe this should be up here. Fixed in ffb0a839fee97a3e3eb2bf7596e329fc451758ee